contracts/anchors/bridged/LinkableCompTokenAnchorPoseidon2.sol

```diff
@@ -20,10 +20,18 @@ contract LinkableCompTokenAnchorPoseidon2 is LinkableAnchorPoseidon2 {
     uint256 _denomination,
     uint32 _merkleTreeHeight,
     uint32 _chainID,
-    IMintableCompToken _token
+    TokenWrapper _token
```

Comment:
It's always more optimised to use interfaces rather than importing the token contract directly (due to bytecode size). If you think we should use the TokenWrapper as the base then I would consider renaming `IMintableCompToken` or creating a new `ITokenWrapper` that handles this.

Reply:
I will create a TokenWrapper Interface!